pom.xml

```diff
@@ -149,7 +151,7 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins.workflow</groupId>
             <artifactId>workflow-job</artifactId>
-            <version>2.12.2</version>
+            <version>2.21</version>
```

Comment:
https://github.com/jenkinsci/workflow-job-plugin/pull/93 in 2.18 fixed `Owner.getListener()` to work right away during `onLoad`, which we need here in order to see the message from the pickle rehydration.

Reply:
Heh.  For once a positive side effect?

Reply:
Indeed.